File: despatch.Domain/Models/Default/Request/ContactDataRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace despatch.Domain.Models.Default.Request
{
    public class ContactDataRequest : BaseEntityRequest
    {
        public string DisplayName { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string MiddleName { get; set; }
        public string HonorificPrefix { get; set; }
        public string HonorificSuffix { get; set; }
        public string Nickname { get; set; }
        public string Note { get; set; }
        public string Email { get; set; }
        public DateTime? BirthdayDate { get; set; }

        public ICollection<ContactDataAddressesRequest> Addresses { get; set; }
        public ICollection<ContactDataOrganizationsRequest> Organizations { get; set; }
        public ICollection<ContactDataImagesRequest> Images { get; set; }
        public ICollection<ContactDataPhoneNumbersRequest> PhoneNumbers { get; set; }
        public ICollection<ContactDataUrlsRequest> Urls { get; set; }
    }
}

File: despatch.WEB/app/src/swagger/models/contact-data-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { ContactDataAddressesRequest } from './contact-data-addresses-request';
import { ContactDataImagesRequest } from './contact-data-images-request';
import { ContactDataOrganizationsRequest } from './contact-data-organizations-request';
import { ContactDataPhoneNumbersRequest } from './contact-data-phone-numbers-request';
import { ContactDataUrlsRequest } from './contact-data-urls-request';
export interface ContactDataRequest extends BaseEntityRequest{
  addresses?: Array<ContactDataAddressesRequest>;
  birthdayDate?: string;
  displayName?: string;
  email?: string;
  firstName?: string;
  honorificPrefix?: string;
  honorificSuffix?: string;
  images?: Array<ContactDataImagesRequest>;
  lastName?: string;
  middleName?: string;
  nickname?: string;
  note?: string;
  organizations?: Array<ContactDataOrganizationsRequest>;
  phoneNumbers?: Array<ContactDataPhoneNumbersRequest>;
  urls?: Array<ContactDataUrlsRequest>;
}
